pkg/cmd/experimental/buildchain/buildchain.go

```diff
@@ -136,17 +136,31 @@ func NewCmdBuildChain(f *clientcmd.Factory, parentName, name string) *cobra.Comm
 				checkErr(err)
 
 				// Validate the specified image repository
-				imgRepo, err := osc.ImageRepositories(namespace).Get(name)
+				_, err = osc.ImageRepositories(namespace).Get(name)
 				checkErr(err)
 
+				// Extract tags from build configurations
 				tags := make([]string, 0)
+				emptyTag := false
 				if allTags {
-					for tag := range imgRepo.Tags {
-						tags = append(tags, tag)
```

Comment:
ImageRepo.Tags is not going to show you all tags on a Repo.  You should be looking at Status.Tags